frontend/Caddyfile

```diff
@@ -14,11 +14,17 @@
 		}
 		level {$LOG_LEVEL}
 	}
+    handle /env.js {
+        header {
+          Content-Type text/javascript
+        }
+        respond `window.config={"VITE_AMS_URL":"{$AMS_URL}"};`
+    }
 	root * /srv
 	encode zstd gzip
 	file_server
 	@spa_router {
-		not path /api/*
+		not path /api/* /env.js
```

Comment:
Is this meant to be `/env.ts`? Or does that file get compiled to `env.js`?

Reply:
the /env.js is a new route which will browser will call out, I will add the missing code :).